index.html

```diff
@@ -396,6 +400,10 @@ <h1>Important Terms</h1>
 			<dd>
 			  <p>An 'owning element' is any <abbr title="Document Object Model">DOM</abbr> ancestor of the <a>element</a>, or any element with an <pref>aria-owns</pref> attribute which references the ID of the   element. </p>
 			</dd>
+			<dt><dfn data-export="" data-lt="parent element|parent|owning parent">Parent Element</dfn></dt>
+			<dd>
+				<p>An owning parent element is a direct <abbr title="Document Object Model">DOM</abbr> parent of the <a>element</a>, an ancestor of the <a>element</a> with only <rref>generic</rref> or <rref>presentation</rref> elements intervening, or any element with a child specified via <pref>aria-owns</pref>.</p>
```

Comment:
the `dt` is for "parent element", but this line references "owning parent element"

Reply:
Updated to "owning parent" to match terms.